patches/chromium/reland_set_app_container_acls_reliably_on_out_directory.patch

```diff
@@ -0,0 +1,80 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Stefan Smolen <ssmole@microsoft.com>
+Date: Tue, 10 Dec 2024 19:38:18 +0000
+Subject: [reland] Set app container ACLs reliably on out directory
+
+This is a reland of:
```

Comment:
The patch is a partial cherry-pick of the reland, the upstream version in this roll only contains https://chromium-review.googlesource.com/c/chromium/src/+/6068009 which leads to failure in cross-compile mode. It was later reverted and relanded in https://chromium-review.googlesource.com/c/chromium/src/+/6081378 with the required `host_os` checks. I didn't want to fast forward roll so that we can merge the PR today. The patch will be removed in the next roll.